### ISSUE TITLE
no-multi-spaces false report on multiple spaces inside template literal

### ISSUE THREAD

**Issue by james-love** [blakeembrey/tslint-config-standard#47]
Ran into an issue where multiple spaces in a template literal will falsely reports [no-multi-spaces](https://eslint.org/docs/rules/no-multi-spaces). Here is an example:
```javascript
const test = `${'test'}  ` // Note the two spaces after ${'test'}
```

I traced it down to an issue with the [tslint-eslint-rules dependency](https://github.com/buzinas/tslint-eslint-rules/issues/313) which has been fixed by the latest version of the package. This should be able to be fixed by updating the tslint-eslint-rules package.

**Comment by blakeembrey** [comment]
Thanks for a heads up, working on a new release now 👍

**Comment by james-love** [comment]
I've updated to the release you made and I'm no longer encountering the error, thank you!

**Comment by blakeembrey** [comment]
@james-love Thanks 😄
